.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,45 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+'
+    
+env: 
+  IMAGE_NAME: samples/event-driven
```

Comment:
`asa-samples/event-driven` ?

Reply:
the image name will be like `unlisted/azurespringapps/samples/hello-world` in mcr, will it be weird if we call it `unlisted/azurespringapps/asa-samples/hello-world`?